client/src/app/modules/project-page/components/header.component.ts

```diff
@@ -27,4 +28,8 @@ export class ProjectHeaderComponent implements OnInit {
   emitOpenNotification(text) {
     this.onOpenNotification.emit(text);
   }
+
+  isEditingPossible(): boolean {
+    return this.hasPlatformRuntime && !this.isQuickstartersError;
```

Comment:
Is ``this.hasPlatformRuntime`` the same as ``this.project.platformRuntime``? In this case maybe the new ``@Input`` isn't needed.